app/app.go

```diff
@@ -697,7 +698,19 @@ func New(
 	app.UpgradeKeeper.SetUpgradeHandler(plan0_7_0, func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
```

Comment:
lets create another upgrade plan?

Reply:
I converted to draft for now, until we made a decision on this issue.

Reply:
Actually, I think it's fine because it'll be released in different versions of binaries.